src/gtc/unstructured/usid.py

```diff
@@ -79,44 +79,11 @@ class VarAccess(Expr):
     ]  # to distinguish from FieldAccess, see https://github.com/eth-cscs/eve_toolchain/issues/34
 
 
-class AssignStmt(Stmt):
-    left: Union[FieldAccess, VarAccess]
-    right: Expr
+AssignStmt = common.AssignStmt[Union[FieldAccess, VarAccess], Expr]
```

Comment:
Alternative
```suggestion
class AssignStmt(common.AssignStmt[Union[FieldAccess, VarAccess], Expr]):
    pass
```
and allows dialect specific validator